WEB_SERVICE/WEB/_SECURE/SECURE_AUTHENTICATION/meta.json

```diff
@@ -7,13 +7,5 @@
   },
   "title": "Enforcer proper authentication",
   "privacy_issue": true,
-  "security_issue": true,
```

Comment:
Was this deleted by mistake?

Reply:
No it wasn't. It’s not intended to have these categories, it is secure.